src/components/seo.js

```diff
@@ -3,14 +3,15 @@ import PropTypes from 'prop-types'
 import Helmet from 'react-helmet'
 import { StaticQuery, graphql } from 'gatsby'
 
-function SEO({ description, lang, image, meta, keywords, title }) {
+function SEO({ description, lang, image, meta, keywords, title, url }) {
   return (
     <StaticQuery
       query={detailsQuery}
       render={data => {
         const metaDescription =
           description || data.site.siteMetadata.description
         const metaImage = image && image.src ? `${data.site.siteMetadata.siteUrl}${image.src}` : null
+        const metaUrl = url || data.site.siteMetadata.siteUrl
```

Comment:
I was going to add this... what if we add the pathname here and prefix with the siteUrl?

I wanted to show a simple and easy way to compose this, so I want to keep it relatively simple and that might complicate things.

Reply:
https://github.com/LekoArts/gatsby-starter-prismic/blob/master/src/components/SEO/SEO.jsx#L37

Yeah, sure, we can also do it like that there. Wasn't sure what you wanted to do

Reply:
Not being prescriptive at all! What do you think is better from a user's perspective?

Reply:
pathname is probably better. Then we can add that to the template (and it happens automatically) and for custom usage you don't have to change the base url everywhere then if you change it.

Reply:
With or without trailing slash then?

Reply:
I think I'd probably create a `pathname` prop to `seo` (I don't really care what this is called... ?) and default it to ``. Then when it's passed it will just include the `/` for the pathname.